.github/workflows/main.yml

```diff
@@ -74,6 +74,9 @@ jobs:
     env:
       TOXENV: "unit"
       PYTEST_ADDOPTS: "-v --color=yes --csv unit_results.csv"
+      DBT_TEST_USER_1: dbt_test_user_1
+      DBT_TEST_USER_2: dbt_test_user_2
+      DBT_TEST_USER_3: dbt_test_user_3
```

Comment:
Oh right - this is fine. We don't need these as repo secrets after all

Reply:
I moved these, but I do think we might want to keep all of the test setup environment variables in the same place, such as POSTGRES_TEST_DATABASE and POSTGRES_TEST_USER.

Reply:
Apparently we didn't actually use environment variables for postgres, just the defaults.